apps/demo/src/app/pages/notification/examples/03-close-notification/index.ts

```diff
@@ -0,0 +1,37 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {NotificationService} from '@ng-web-apis/notification';
+import {PermissionsService} from '@ng-web-apis/permissions';
+import {timer} from 'rxjs';
+import {filter, switchMap, takeUntil} from 'rxjs/operators';
+
+@Component({
+    selector: 'notification-page-example-3',
+    templateUrl: './index.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NotificationPageExample3 {
+    readonly notificationPermissionState$ =
```

Comment:
If we add mapper utils we can just rename this as just `denied$` in all of the examples which would also make look cleaner.

Reply:
✅